src/Mechanical/Rotational/sources.jl

```diff
@@ -2,11 +2,61 @@
     Torque(;name) 
 
 Input signal acting as external torque on a flange
+
+# States:
+- See [PartialElementaryOneFlangeAndSupport2](@ref)
+
+# Connectors:
+- See [PartialElementaryOneFlangeAndSupport2](@ref)
+- `tau` [RealInput](@ref) input for torque control signal
+
+# Parameters:
+- `use_support`: If support flange enabled, otherwise implicitly grounded
 """
 function Torque(;name, use_support=false) 
     @named partial_element = PartialElementaryOneFlangeAndSupport2(use_support=use_support)
     @unpack flange = partial_element
     @named tau = RealInput() # Accelerating torque acting at flange (= -flange.tau)
     eqs = [flange.tau ~ -tau.u]
     return extend(ODESystem(eqs, t, [], []; name=name, systems=[tau]), partial_element)
+end
+
+
+"""
+    Speed(;name) 
+
+Forced movement of a flange according to a reference angular velocity signal
+
+# States:
+- See [PartialElementaryOneFlangeAndSupport2](@ref)
+
+# Connectors:
+- See [PartialElementaryOneFlangeAndSupport2](@ref)
+- `w_ref` [RealInput](@ref) Reference angular velocity of flange with respect to support as input signal
+
+# Parameters:
+- `use_support`: If support flange enabled, otherwise implicitly grounded
+- `exact`: true/false exact treatment/filtering the input signal
+- `f_crit`: [Hz] if exact=false, critical frequency of filter to filter input signal 
+"""
+function Speed(;name, use_support=false, exact=false, f_crit=50) 
```

Comment:
`f_crit=50` seems somewhat arbitrary (chosen with electrical grid frequency in mind?). A user incorporating `Speed()` in their model may have a hard-to find bug due to this default value, especially since filtering is enabled by default.

Reply:
I took the default from [Modelica](https://doc.modelica.org/Modelica%204.0.0/Resources/helpWSM/Modelica/Modelica.Mechanics.Rotational.Sources.Speed.html), but you are right more documentation is needed.

Reply:
I think it is not bad to filter by default, otherwise the user would be only allowed to supply continuously differentiable inputs.

Reply:
filter by default seems alright, my problem is more with the default filter frequency which the user might not expect. If there is no default, the user will have to supply it which I find less bug prone

Reply:
Thinking a bit more about this, 50Hz is a particularly bad choice for a default filter cutoff frequency. It is just slightly below the frequency of the US grid and barely touches the noise form the European grid. Defaults make sense if they are correct for most people, 50Hz is likely correct for next to no people.

Reply:
I think this has nothing to do with el. grid frequency. After all it is the input for a mechanical system. But we can remove the default value. Maybe it is easier to specify a time constant than a frequency. It is a first order filter hence most users will have an idea of what the step response would look like.

Reply:
Ah, I'm confusing it with `SpeedSensor`.
> Maybe it is easier to specify a time constant than a frequency.

Yeah that's perhaps a good idea, when it's a frequency it feels very much like we're specifying a noise filter.